test/test.js

```diff
@@ -56,7 +56,7 @@ describe('rollup-plugin-babel', function() {
 				Object.assign(
 					{
 						input,
-						plugins: [babelPlugin(babelOptions)],
+						plugins: [babelPlugin(Object.assign({ babelHelpers: 'bundled' }, babelOptions))],
```

Comment:
```suggestion
						plugins: [babelPlugin({ babelHelpers: 'bundled', ...babelOptions })],
```

Reply:
This plugin is bundled by buble instead of babel (couldnt figure out building with itself) and I had some issues with transforming spread operator with buble, but gonna give it a try.

Reply:
:joy: self-hosting is always fun

Reply:
Yeah looks like this is because `buble/register` doesn't support options.